### ISSUE TITLE
Help attribute

### ISSUE THREAD

**Issue by napulen** [DDMAL/MEI2Volpiano#10]
https://github.com/DDMAL/MEI2Volpiano/blob/b785992972d896d8d63a328b85ebdbca961dc1a7/mei2volpiano.py#L100

The `help` parameter is intended as a plain English description of what is expected of this parameter. See example in the [argparse](https://docs.python.org/3/library/argparse.html) documentation:

![image](https://user-images.githubusercontent.com/7258463/117972195-ce6fd080-b2f8-11eb-99ed-1c19b89ac3bc.png)

It's not really an error. By the way, `argparse` will generate error messages and `usage` messages automatically for you, so you can rely on those features.

Additionally, if the variable has a string, no need to format it, you can refer to it directly in the parameters sent to argparse:

```python
parser.add_argument("mei_files", type=str, nargs="+", help=error)
```

In short:
- I think calling it `error` is misleading. This is/should be a description.
- No need to use an f-string here

**Comment by kemalkongar** [comment]
Argparse section has been redone and `error` clarified.
